pkg/model/manifest.go

```diff
@@ -56,8 +144,271 @@ type DeployInfo struct {
 	Commands []string `json:"commands,omitempty" yaml:"commands,omitempty"`
 }
 
+//NewDeployInfo creates a deploy Info
 func NewDeployInfo() *DeployInfo {
 	return &DeployInfo{
-		Commands: make([]string, 0),
+		Commands: []string{},
+	}
+}
+
+//GetManifestV2 gets a manifest from a path or search for the files to generate it
+func GetManifestV2(manifestPath string) (*Manifest, error) {
```

Comment:
if `okteto.yml` is the new manifest, does it take precedence over the rest of manifests?

Reply:
Added